src/popover/src/Popover.js

```diff
@@ -25,6 +25,7 @@ export default class Popover extends Component {
      * When true, the Popover is manually shown.
      */
     isShown: PropTypes.bool,
+    trigger: PropTypes.oneOf(['click', 'hover']),
```

Comment:
Could you add a jsdoc description for the prop here?

Reply:
done!

Reply:
Oh, that reminds me – maybe we should add `trigger: 'click'` to `defaultProps`? I realize with the current implementation it's not strictly necessary, but would be good for semantics I think.

Reply:
done